### PR TITLE
Update edgekv.js with fire and forget write operations.

### PR DESCRIPTION
**Pull request by kayoub-akamai**
Add non-async methods to PUT data in EdgeKV that will not wait for a response from origin. This will provide instant write operations within the EdgeWorker.

### CODE REVIEW

File: edgekv/lib/readme.md [hunk 1]
```diff
@@ -113,6 +135,15 @@ The edgekv.js file must be included alongside `edgekv_tokens.js` which includes
 	} catch (error) {
 		// do something in case of an error
 	}
+### Write the Product Modification Time Without Wait For Response
```
Comment (mryakan) [inline edgekv/lib/readme.md:138]: Mention in the title that this is an example, or possibly put under an examples section?

Comment (kayoub-akamai) [reply edgekv/lib/readme.md:138]: It is in the examples section.

Comment (mryakan) [reply edgekv/lib/readme.md:138]: my bad, the PR is not showing whole file for some reason. I'll merge it in.

**Comment by mryakan** [comment]
The EdgeKV helper library has been upgraded to include non-async methods to PUT data into EdgeKV. This provides the ability to write data without waiting for a response to complete the operation.

**Pull request merged**